docs/src/TextArea.doc.js

```diff
@@ -32,12 +32,6 @@ card(
         type: 'string',
         required: true,
       },
-      {
-        name: 'idealErrorDirection',
```

Comment:
What's the migration strategy for removing these props?

Reply:
Not sure. What would you recommend? The removed prop refers to a feature we no longer offer (direction of error), so they seem superfluous but not harmful?

Reply:
@dangerismycat in reference to @chrislloyd's point can we include a codemod with this PR (or a quick followup) that will remove the prop from our codebase? if we dont then this bump will cause flow errors everywhere the component prop is used

Reply:
👍